inscriptions/ISic003869.xml

```diff
@@ -182,21 +184,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI334306">
-                <ab>
-                    <lb n="1"/>
-                    <unclear>Κ</unclear>άλλιος.
-                    <lb n="2"/>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#SS">
+                <ab> <!-- the edition calls for recto and verso, the recto being the more recent inscription; not working here, as the recto text has its own record and file -->
+                    <lb n="1"/><unclear>Κ</unclear>άλλιος
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <app loc="1">
+                    <rdg>ΚΑΛΛΙΟΣ</rdg>
```

Comment:
we need to have a serious talk about apparatus at some point, as we're not using it consistently or systematically and it would be good to agree the best way forward here.